csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -790,7 +790,28 @@ private void TestMultiThreads()
                 { "test_training_dropout_default_mask", "node test error"},
                 { "test_min_int8", "node test error"},
                 { "test_cast_FLOAT_to_STRING", "node test error"},
-                { "test_identity_sequence", "data type not supported"}
+                { "test_identity_sequence", "data type not supported"},
+                { "test_gru_batchwise"},
+                { "test_gru_defaults"}, 
+                { "test_gru_seq_length"}, 
+                { "test_gru_with_initial_bias"},
+                { "test_identity_sequence"},
+                { "test_lstm_batchwise"},
+                { "test_lstm_defaults"},
+                { "test_lstm_with_initial_bias"},
+                { "test_lstm_with_peepholes"},
+                { "test_rnn_seq_length"},
+                { "test_simple_rnn_batchwise"},
+                { "test_simple_rnn_defaults"},
+                { "test_simple_rnn_with_initial_bias"},
+                { "test_sub_uint8"},
+                { "test_mul_uint8"},
+                { "test_add_uint8"},
+                { "test_div_uint8"},
+                { "test_batchnorm_epsilon"},
+                { "test_batchnorm_epsilon_trining_mode"},
```

Comment:
typos: "trining" and "trianing"
do the actual test names have these too?

Reply:
no... it was my typo... fixed it